MapboxCoreNavigationTests/LocationTests.swift

```diff
@@ -31,4 +40,43 @@ class LocationTests: XCTestCase {
         XCTAssert(lhs == rhs)
     }
     
+    func testSnappedLocation100MetersAlongRoute() {
+        let progress = setup.progress
+        let firstLocation = setup.firstLocation
+        
+        let initialHeadingOnFirstStep = progress.currentLegProgress.currentStep.finalHeading!
+        let coordinateAlongFirstStep = firstLocation.coordinate.coordinate(at: 100, facing: initialHeadingOnFirstStep)
+        let locationAlongFirstStep = CLLocation(latitude: coordinateAlongFirstStep.latitude, longitude: coordinateAlongFirstStep.longitude)
+        guard let snapped = locationAlongFirstStep.snapped(to: progress.currentLegProgress) else {
+            return XCTFail("Location should of snapped to route")
```

Comment:
`Location should ?? of ?? snapped to route` -> `Location should have snapped to route` -- maybe?

Reply:
Great catch, @vincethecoder!